examples/pubsub/server_pubsub_subscribe_monitoring_linux.c

```diff
@@ -0,0 +1,539 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information. */
```

Comment:
My proposal: Make a subfolder in the pubsub example folder called 'monitoring'. Then move this file there and call it ..._custom_monitoring instead of monitoring_linux.